Modules/_collectionsmodule.c

```diff
@@ -393,16 +398,18 @@ deque_appendleft_internal(dequeobject *deque, PyObject *item, Py_ssize_t maxlen)
     Py_SET_SIZE(deque, Py_SIZE(deque) + 1);
     deque->leftindex--;
     deque->leftblock->data[deque->leftindex] = item;
-    if (NEEDS_TRIM(deque, deque->maxlen)) {
+    if (NEEDS_TRIM(deque, maxlen)) {
         PyObject *olditem = deque_pop_impl(deque);
         Py_DECREF(olditem);
-    } else {
+    }
+    else {
```

Comment:
```suggestion
    } else {
```

Reply:
According to PEP7, the current style is correct.
https://peps.python.org/pep-0007/

Reply:
Ah you just wanted to revert style things. Got it.